dashboard/src/components/DeploymentForm/DeploymentForm.test.tsx

```diff
@@ -190,6 +167,12 @@ it("triggers a deployment when submitting the form", async () => {
   act(() => {
     handleValuesChange("foo: bar");
   });
+
+  // Set the release name
+  const releaseNameInput = wrapper.find("#releaseName");
+  releaseNameInput.getDOMNode<HTMLInputElement>().value = releaseName;
+  releaseNameInput.simulate("change");
```

Comment:
fyi, you can do, `.simulate("change", {target: {value: releaseName}})` (you may need to change `currentTarget` with `target` in the code though.

Reply:
Done! I wasn't sure if there were any side effects of changing `currentTarget` with `target` so I implemented this way. But, I've checked it and everything goes fine, so the less wordy the better.
Thanks for the tip!